yarn.lock

```diff
@@ -3574,6 +3574,11 @@
   version "3.0.3"
```

Comment:
Removes nothing, adds a whole bunch? I'm sure it's okay but it's a bit unexpected

Reply:
A lot of dev deps are added because of the legacy-polyfills package.